docs/features/logforwarding/outputs/splunk-forwarding.adoc

```diff
@@ -80,10 +27,12 @@ spec:
     - name: splunk-receiver
       type: splunk
       splunk:
-       indexKey: "kubernetes.namespace_name"
-      secret:
-        name: splunk-secret
-      url: 'http://example-splunk-hec-service:8088'
+        authentication:
+          token:
+            key: hecToken
+            secretName: splunk-secret
+        url: 'http://example-splunk-hec-service:8088'
+        index: "{.log_type | 'undefined'}"
```

Comment:
Double pipe `||` but also static value must be encased in double quotes `""`. Use `'` instead of double quotes.
```suggestion
        index: '{.log_type || "undefined"}'
```